types/howler/index.d.ts

```diff
@@ -208,7 +208,9 @@ export class Howl {
     rate(idOrSetRate: number): this | number;
     rate(rate: number, id: number): this;
 
-    seek(seek?: number, id?: number): this | number;
+    seek(): number;
+    seek(idOrSetSeek: number): this | number;
```

Comment:
What is `idOrSetSeek`? Why does this return `this` or a `number`?  In fact, they should all return a single type, not a union. Or document why it's not for `seek(idOrSetSeek)`.

Reply:
here is [document ](https://github.com/goldfire/howler.js#seekseek-id) of the seek function

here is [codesandbox](https://codesandbox.io/s/epic-agnesi-j2mjq?file=/src/index.js) shows the return type of seek function

Reply:
The documentation does not describe this. Only the code does. Please comment this.

Also, please add tests.